mini/html+css/helloworld.html

```diff
@@ -0,0 +1,64 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="utf-8">
+  <meta http-equiv="x-ua-compatible" content="ie=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
+  <title>Hello World</title>
+  <link rel="stylesheet" href="helloworld.css">
+</head>
+<body>
+  
+  <div id="navbar">
+	  <div class="links">
+	  	<ul>
+	  		<li class="nav">Navbar</li>
+			<li><a href="#" role="button" aria-pressed="true">Home</a></li>
+			<li><a href="#" role="button" aria-pressed="true">Link</a></li>
+			<li><a href="#" role="button" aria-pressed="true">Link2</a></li>
+			<li><a href="#" role="button" aria-pressed="true">Link3</a></li>
+	  	</ul>
+	  </div>
+	  
+	  <div class="search">
+	  	<input type="text" name="search" value="Search" class="search1">
+		<input type="button" name="button" value="Submit" class="submit">
+	  </div>
+  </div>
+  
+  <div class="welcome">
+	  <div class="content">
+	  	<h1>Hello, world!</h1>
+		<p>Lorem ipsum is a pseudo-Latin text used in web design, typography, layout, and printing in place of English to emphasise design elements over content. It's also called placeholder (or filler) text. It's a convenient tool for mock-ups. It helps to outline the visual elements of a document or presentation, eg typography, font, or layout. Lorem ipsum is mostly a part of a Latin text by the classical author and philosopher Cicero. Its words and letters have been changed by addition or removal, so to deliberately render its content nonsensical; it's not genuine, correct, or comprehensible Latin anymore. While lorem ipsum's still </p>
+		<button type="button" class="button1">Learn More</button>
+	  </div>
+  	
+  </div>
+  
+  <div class="main">
+	  <div class="subbox1">
```

Comment:
case style for subbox?
repeat lines 45 and 51

Reply:
not inlcuded in this project. it was part of the last one

Reply:
included*